tcms/bugs/tests/test_views.py

```diff
@@ -0,0 +1,62 @@
+# pylint: disable=too-many-ancestors
+from django.urls import reverse
+from django.utils.translation import gettext_lazy as _
+
+from tcms.tests import BaseCaseRun, user_should_have_perm
+from tcms.tests.factories import BugFactory
+from tcms.utils.permissions import initiate_user_with_default_setups
+from tcms.core.templatetags.extra_filters import markdown2html as md2html
+
+
+class TestBugStatusChange(BaseCaseRun):
+    """Test the possible bug status changes.
+
+    Cases:
+    * Closing an open bug.
+    * Reopening a closed bug.
+    """
+
+    @classmethod
+    def setUpTestData(cls):
+        super().setUpTestData()
+
+        initiate_user_with_default_setups(cls.tester)
+        cls.comment_bug_url = reverse('bugs-comment')
+        user_should_have_perm(cls.tester, 'bugs.change_bug')
+
+    def test_close_an_open_bug(self):
+        bug = BugFactory(status=True)
+
+        edit_bug_data = {
+            'bug': bug.pk,
+            'text': 'Close the bug.',
+            'action': 'close'
+        }
+
+        redirect_url = reverse('bugs-get', args=[bug.pk])
+        response = self.client.post(self.comment_bug_url, edit_bug_data, follow=True)
+
+        self.assertContains(response, md2html(_('*bug closed*')))
+        self.assertContains(response, _('Close the bug.'))
```

Comment:
This is the text coming from the comment form. It isn't translated so it doesn't make sense to try and translate it while searching for it.